### PR TITLE
1476 radio group doesnt support custom values

### LINKED ISSUE #1476: RadioGroup doesn't support custom values
RadioGroup supports only options with string values.
It should be a generic component instead, supporting values of any type.

### PR DESCRIPTION
**Pull request by veej**
Closes #1476 

### Test performed
image

### CODE REVIEW

File: src/DatePicker/datePicker.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 @import '../utils/theme/variables.scss';
-@import 'react-dates/lib/css/_datepicker';
+@import '~react-dates/lib/css/_datepicker.css';
```
Comment (giogonzo) [inline src/DatePicker/datePicker.scss:2]: related?

File: src/RadioGroup/RadioGroup.tsx [hunk 2]
```diff
@@ -2,18 +2,19 @@ import * as React from 'react';
 import * as cx from 'classnames';
 import FlexView from 'react-flexview';
 
-export type RadioOption = {
+export type RadioOption<T = string> = {
```
Comment (giogonzo) [inline src/RadioGroup/RadioGroup.tsx:5]: do we need the `= default` at all?

File: src/DatePicker/datePicker.scss (hunk 1 above)
Comment (veej) [reply src/DatePicker/datePicker.scss:2]: I had to rebase on https://github.com/buildo/react-components/pull/1474 (not yet merged) to make it compile.

File: src/RadioGroup/RadioGroup.tsx (hunk 2 above)
Comment (veej) [reply src/RadioGroup/RadioGroup.tsx:5]: maybe not

**Review by giogonzo — APPROVED** [review]

**Pull request merged**